team_admin/user_accounts.md

```diff
@@ -1,4 +1,5 @@
 ---
+redirect_from: /user_accounts
```

Comment:
Urgh, shame there's not a way to issue a proper redirect rather than using HTML `<meta>`.